outputs.tf

```diff
@@ -1,21 +1,26 @@
-output "ecs_cluster" {
-  value = local.create_new_cluster ? aws_ecs_cluster.new_cluster[0] : data.aws_ecs_cluster.existing_cluster[0]
+output "new_ecs_cluster" {
+  value = local.create_new_cluster ? aws_ecs_cluster.new_cluster[0] : null
 }
 
 output "fargate_security_group" {
-  value = aws_security_group.fargate_service_sg
+  value = aws_security_group.fargate_sg
 }
 
 output "task_definition" {
   value = aws_ecs_task_definition.task_def
 }
 
+output "schedule" {
+  value     = local.use_scheduler ? aws_scheduler_schedule.schedule[0] : null
+  sensitive = true
```

Comment:
Nit: Why is this marked sensitive?

Reply:
Because terraform yelled at me when it wasn't 😅 I don't know why a scheduler_schedule has sensitive values, not even sure which ones